commands/cfe.js

```diff
@@ -3,6 +3,23 @@
 const Validation = require('../utils/validator');
 const myValidator = new Validation();
 const sendMessage = require('../utils/sendTableMessage');
+const Scrapper = require('../webScraping/cfeScrapper');
+
+/**
+ * Do the scrap
+ * @param { string } args Search parameter
+ * @param { string } route Where the file will be saved
+ */
+async function makeScrap(args, route) {
+  const cfeScrapper = new Scrapper(args);
+  const scrap = await cfeScrapper.doScraping(route);
+  if (scrap === false) {
```

Comment:
what this mean ? what should return the doScraping ??

Reply:
If there's no data available, because of the parameter doesn't have results in the table, doScraping will return a false